### PR TITLE
Remove .travis.yml and add Github Actions

### PR DESCRIPTION
**Pull request by JakobGray**
Part of move to get off Travis. Jobs that were previously run in Travis are now handled by Github Action workflows. This PR will run chart linting on PRs, fast-forward the main branch to release-X.Y, and send a dispatch to multicloudhub-repo to pick up changes to the current release branch.

### CODE REVIEW

File: .github/workflows/push-chart.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+  notify-repo:
+    runs-on: ubuntu-latest
+    steps:
+      # Sends a dispatch event to multicloudhub-repo to trigger an update
+      - name: Notify Repo
+        uses: peter-evans/repository-dispatch@v1.1.3
+        with:
+          token: ${{ secrets.HUB_REPO_TOKEN }}
```
Comment (jlpadilla) [inline .github/workflows/push-chart.yml:20]: Where are these secrets configured?

**Review by jlpadilla — APPROVED** [review]
Have a question about where the secrets come from. Otherwise good to merge.

File: .github/workflows/push-chart.yml (hunk 1 above)
Comment (JakobGray) [reply .github/workflows/push-chart.yml:20]: They're defined in https://github.com/open-cluster-management/search-chart/settings/secrets/actions. You're welcome to update the FF_TOKEN with a token of your own if you'd like, as long as it has enough privilege to push to a release branch

Comment (JakobGray) [reply .github/workflows/push-chart.yml:20]: I created them using the acm-installer bot account so it wouldn't need to be tied to anyone's personal account

Comment (jlpadilla) [reply .github/workflows/push-chart.yml:20]: Thanks! Merging.

**Pull request merged**